src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/X86/Ssse3.PlatformNotSupported.cs

```diff
@@ -11,10 +11,10 @@ namespace System.Runtime.Intrinsics.X86
     /// This class provides access to Intel SSSE3 hardware instructions via intrinsics
     /// </summary>
     [CLSCompliant(false)]
-    public static class Ssse3
+    public abstract class Ssse3 : Sse2
```

Comment:
12.7.2 Checking for SSSE3 Support

Before an application attempts to use the SSSE3 extensions, the application should follow the steps illustrated in
Section 11.6.2, “Checking for SSE/SSE2 Support.” Next, use the additional step provided below:

• Check that the processor supports SSSE3 (if CPUID.01H:ECX.SSSE3[bit 9] = 1).

Reply:
Confirmed with Intel that `Ssse3` requires `Sse3`, despite that it is not explicitly listed.

CC. @fiigii

Reply:
12.7.1 Guidelines for Using SSSE3 Extensions
The following guidelines describe how to maximize the benefits of using SSSE3 extensions:
• Check that the processor supports SSSE3 extensions.
• Ensure that your operating system supports SSE/SSE2/SSE3/SSSE3 extensions. (Operating system support
for the SSE extensions implies sufficient support for SSE2, SSE3, and SSSE3.)
• Employ the optimization and scheduling techniques described in the Intel® 64 and IA-32 Architectures Optimization
Reference Manual (see Section 1.4, “Related Literature”).